WooCommerce/Classes/View Modifiers/WooStyleModifiers.swift

```diff
@@ -86,6 +86,13 @@ struct ErrorStyle: ViewModifier {
     }
 }
 
+struct NavigationBarStyle: ViewModifier {
+    func body(content: Content) -> some View {
+        content
+            .accentColor(Color(.accent)) // The color of bar button items in the navigation bar
```

Comment:
I think someone mentioned that we have to use `tint` for iOS 15, right?

Reply:
Eventually we'll want to add `tint`, but after trying it out I realized we'll have to wait on this. `tint` isn't available in Xcode 12.5, which we're still using for CI. I checked and `accentColor` does still work on iOS 15.0, so for now we can use it.

Reply:
Would it make sense to leave a comment with some sort of reminder about that upcoming change?